options_test.go

```diff
@@ -0,0 +1,55 @@
+package leaseweb
```

Comment:
This test should be in the `options` package without relying on `DedicatedServerListOptions`

Reply:
DONE